youtube_dl/extractor/twitter.py

```diff
@@ -229,7 +229,10 @@ def _real_extract(self, url):
 
         title = self._search_regex(r'<title>([^<]+)</title>', webpage, 'title')
         thumbnail = config.get('posterImageUrl') or config.get('image_src')
-        duration = float_or_none(config.get('duration')) or duration
+        if config.get('duration'):
+            duration = float_or_none(config.get(duration))
+            if duration:
+                duration *= 10e-4
```

Comment:
1. Do not remove fallback to previous duration value.
2. `scale` of `float_or_none` instead.